src/scss/skin-modern/_skin-tv.scss

```
@@ -16,7 +16,8 @@
     }
   }
 
-  .#{$prefix}-ui-titlebar-top {
+  .#{$prefix}-ui-titlebar-top,
+  .#{$prefix}-ui-menucaption {
```

I guess we don't need this anymore, right?
```suggestion
  .#{$prefix}-ui-titlebar-top {
```

---

Good catch, done in 5efe823e